assessments/array-assessment.js

```diff
@@ -15,24 +15,25 @@ arrayOfStrings.forEach(item => console.log(item));
 // 2.
 // For arrayOfNumbers, return a new array where the values are multiplied by 10
 // expected output: [50, 40, 30...]
-
+arrayOfNumbers.forEach(mult => console.log(mult * 10));
 
 // 3.
 // Find all of the values in arrayOfNumbers that are > 5
-
+arrayOfNumbers.forEach(great => console.log(great > 5));
```

Comment:
To find all items that match a condition, use `.filter`

```
arrayOfNumbers.filter(number => number > 5)
```

This will only select items from the array that are > 5.